docs/cpp/source/Doxyfile

```diff
@@ -74,7 +74,7 @@ CREATE_SUBDIRS         = NO
 FULL_PATH_NAMES        = YES
 # Nested folders will be ignored without this.
 RECURSIVE              = YES
-# Blacklist certain file patterns from the INPUT section.
+# Block certain file patterns from the INPUT section.
```

Comment:
What do you think about 

```suggestion
# Exclude certain file patterns from the INPUT section.
```

as the line below suggests?

Reply:
Sounds good!